docs/RBACAPI.mdx

```diff
@@ -8,6 +8,19 @@ import TabItem from '@theme/TabItem';
 
 A more friendly API for RBAC. This API is a subset of Management API. The RBAC users could use this API to simplify the code.
 
+> Note: Conventionally subject token name in policy definition is `sub` and placed in the beginning. Now Golang version supports customized token name & place. If the subject token name is `sub`, the subject token can be placed at an arbitrary place and no extra action needs.  If the subject token name is other than `sub` , `e.SetFieldIndex()` for `constant.SubjectIndex` should be called after the enforcer is initialized.
+>
```

Comment:
Perhaps a note should also be added to Model - RBAC directing the user to look at this because the probability of users looking at the Model part may be greater than that in the API part

Reply:
Agreed with @tangyang9464

Reply:
Initially, I don't want to add details involving specific API, i.e., SetFieldIndex, in the introduction of the model.

Reply:
done